### ISSUE TITLE
Setting area OSM env variable breaks the preprocessor

### ISSUE THREAD

**Issue by mehdisadeghi** [hiposfer/kamal#226]
```
➜ env | grep -i freiburg
FREIBURG_AREA_GTFS=resources/freiburg.gtfs.zip
FREIBURG_AREA_OSM=resources/freiburg.osm.bz2

kamal on  master [✘!?]
➜ lein run -m hiposfer.kamal.preprocessor resources/
2018-09-17 22:33:10.025:INFO::main: Logging initialized @6096ms
18-09-17 20:33:11 mpro.local INFO [hiposfer.kamal.preprocessor:55] - preprocessing OSM and GTFS files
Exception in thread "main" java.lang.AssertionError: Assert failed: -- Spec failed --------------------

  {:FREIBURG_AREA_GTFS ..., :FREIBURG_AREA_OSM ...}
                            ^^^^^^^^^^^^^^^^^^

should satisfy

  gtfs-area?

-- Relevant specs -------

:hiposfer.kamal.preprocessor/env:
  (clojure.spec.alpha/and
   (clojure.core/fn [%] (clojure.core/pos? (clojure.core/count %)))
   (clojure.spec.alpha/map-of
    hiposfer.kamal.core/area-id?
    clojure.core/string?)
   (clojure.spec.alpha/map-of
    hiposfer.kamal.preprocessor/gtfs-area?
    clojure.core/string?))

-------------------------
Detected 1 error

(s/valid? :hiposfer.kamal.preprocessor/env areas), compiling:(/private/var/folders/kh/42ww8ghx1z30qbn3lnr7q6j4_0hblj/T/form-init540243156959162582.clj:1:124)
	at clojure.lang.Compiler.load(Compiler.java:7526)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: java.lang.AssertionError: Assert failed: -- Spec failed --------------------

  {:FREIBURG_AREA_GTFS ..., :FREIBURG_AREA_OSM ...}
                            ^^^^^^^^^^^^^^^^^^

should satisfy

  gtfs-area?

-- Relevant specs -------

:hiposfer.kamal.preprocessor/env:
  (clojure.spec.alpha/and
   (clojure.core/fn [%] (clojure.core/pos? (clojure.core/count %)))
   (clojure.spec.alpha/map-of
    hiposfer.kamal.core/area-id?
    clojure.core/string?)
   (clojure.spec.alpha/map-of
    hiposfer.kamal.preprocessor/gtfs-area?
    clojure.core/string?))

-------------------------
Detected 1 error

(s/valid? :hiposfer.kamal.preprocessor/env areas)
	at hiposfer.kamal.preprocessor$_main.invokeStatic(preprocessor.clj:58)
	at hiposfer.kamal.preprocessor$_main.invoke(preprocessor.clj:47)
	at clojure.lang.Var.invoke(Var.java:381)
	at user$eval3132.invokeStatic(form-init540243156959162582.clj:1)
	at user$eval3132.invoke(form-init540243156959162582.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	... 12 more

kamal on  master [✘!?] took 10s
➜
```

**Comment by mehdisadeghi** [comment]
And the content of the resources directory:

```kamal on  master [✘!?] took 10s
➜ ls -lh resources
total 397336
-rw-r--r--  1 sadeghime  500061516   3.4K Apr 17 20:19 benchmark.txt
-rw-r--r--  1 sadeghime  500061516   927B Sep  5 00:17 eastwood.clj
-rw-r--r--  1 sadeghime  500061516   9.6M Sep 14 22:07 frankfurt-am-main.edn.gz
-rw-r--r--  1 sadeghime  500061516    14M Sep  5 00:17 frankfurt.gtfs.zip
-rw-r--r--@ 1 sadeghime  500061516   179K Sep 17 22:31 freiburg.gtfs.zip
-rw-r--r--@ 1 sadeghime  500061516   170M Sep 17 22:29 freiburg.osm.bz2
-rw-r--r--@ 1 sadeghime  500061516    26K Sep 17 22:22 gundelfingen.gtfs.zip
-rw-r--r--  1 sadeghime  500061516   2.5K Sep 14 22:07 kamal.postman_collection.json
-rw-r--r--  1 sadeghime  500061516   1.4K Sep 17 22:04 overpass-api-query.txt
```

**Comment by carocad** [comment]
@mehdisadeghi thanks for the extra input. It helped me pinpoint the problem.

```
{:FREIBURG_AREA_GTFS ..., :FREIBURG_AREA_OSM ...}
                            ^^^^^^^^^^^^^^^^^^
```

This is right. kamal no longer needs the OSM key. Now it uses the name that you put in the env var directly as value for osm. For example `Frankfurt_am_Main_AREA_GTFS=....` -> `:area/name "Frankfurt am Main", :area/gtfs ....`

I recently started disliking this approach since it complects many things together but havent have the time to refactor it.

**Comment by carocad** [comment]
this was already solved in #233
